typings/index.d.ts

```diff
@@ -1,14 +1,35 @@
 // Type definitions for react-datetime
 // Project: https://github.com/YouCanBookMe/react-datetime
 // Definitions by: Ivan Verevkin <vereva@x-root.org>
+//     Updates by: Aaron Spaulding <aaron@sachimp.com>
 
-//// <reference path="../moment/moment-node.d.ts" />
+declare module "react-datetime" {
```

Comment:
Here you are using double-quotes but on L17 you are using single-quotes. Does the module declaration require double-quotes? If not I think you should change them to stay consistent :)